### ISSUE TITLE
[v1.12.11] dependency of addon not found

### ISSUE THREAD

**Issue by simonihmig** [ef4/ember-auto-import#111]
This branch of `ember-cli-yadda` https://github.com/albertjan/ember-cli-yadda/pull/79 adding support for `ember-auto-import` works fine with 1.12.10, but fails with 1.12.11:

```
Could not find module `yadda` imported from `ember-cli-yadda/test-support/yadda`
```

**Comment by nickschot** [comment]
Same issue for https://github.com/auth0-community/ember-simple-auth-auth0 . Also broken on 1.12.12.

**Comment by ef4** [comment]
For ember-cli-yadda, there's a combination of two bugs involved. One I fixed in 1.2.13. But the other is a change you need to make in ember-cli-yadda. This line 

https://github.com/albertjan/ember-cli-yadda/blob/b9f233c326b8ada50a052bdc6077e96261e59572/index.js#L41

Needs to change from

```js
this._super.included(app);
```

to

```js
this._super.included.apply(this, arguments);
```

I don't completely understand why ember-cli has this megatroll, but I know that it causes recursive `included` hooks to not run.

I wasn't able to reproduce a separate failure in ember-simple-auth-auth0, please file a separate issue with a reproduction if that turns out to be a different bug.

**Comment by simonihmig** [comment]
@ef4 awesome, thanks for the quick fix and your further investigation here! 👍
